fr.inria.corese.CoreseCommand.yml

```diff
@@ -0,0 +1,69 @@
+app-id: fr.inria.corese.CoreseCommand
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: /app/bin/run.sh
+modules:
+  - name : openjdk
+    buildsystem: simple
+    build-commands:
+    - /usr/lib/sdk/openjdk11/install.sh
+  - name: corese-command
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 corese-command.jar /app/bin/corese-command.jar
+      - install -Dm755 run.sh /app/bin/run.sh
+      - install -Dm644 fr.inria.corese.CoreseCommand.appdata.xml /app/share/metainfo/fr.inria.corese.CoreseCommand.appdata.xml
+      - install -Dm644 fr.inria.corese.CoreseCommand.desktop /app/share/applications/fr.inria.corese.CoreseCommand.desktop
+      - install -Dm644 fr.inria.corese.CoreseCommand-512.png /app/share/icons/hicolor/512x512/apps/fr.inria.corese.CoreseCommand.png
+      - install -Dm644 fr.inria.corese.CoreseCommand-256.png /app/share/icons/hicolor/256x256/apps/fr.inria.corese.CoreseCommand.png
+      - install -Dm644 fr.inria.corese.CoreseCommand-128.png /app/share/icons/hicolor/128x128/apps/fr.inria.corese.CoreseCommand.png
+      - install -Dm644 fr.inria.corese.CoreseCommand-64.png /app/share/icons/hicolor/64x64/apps/fr.inria.corese.CoreseCommand.png
+      - install -Dm644 fr.inria.corese.CoreseCommand-48.png /app/share/icons/hicolor/48x48/apps/fr.inria.corese.CoreseCommand.png
+      - install -Dm644 fr.inria.corese.CoreseCommand-32.png /app/share/icons/hicolor/32x32/apps/fr.inria.corese.CoreseCommand.png
+      - install -Dm644 fr.inria.corese.CoreseCommand-24.png /app/share/icons/hicolor/24x24/apps/fr.inria.corese.CoreseCommand.png
+      - install -Dm644 fr.inria.corese.CoreseCommand-22.png /app/share/icons/hicolor/22x22/apps/fr.inria.corese.CoreseCommand.png
+      - install -Dm644 fr.inria.corese.CoreseCommand-16.png /app/share/icons/hicolor/16x16/apps/fr.inria.corese.CoreseCommand.png
+      - install -Dm644 fr.inria.corese.CoreseCommand.svg /app/share/icons/hicolor/scalable/apps/fr.inria.corese.CoreseCommand.svg
+    sources:
+      - type: file
+        url: https://github.com/Wimmics/corese/releases/download/release-4.4.1/corese-command-4.4.1.jar
+        sha256: 2b15c46a9643eafb1119db9579e4f259e576647d9f322f437089960120960393
+        x-checker-data:
+          - type: json
+          - url: https://api.github.com/repos/wimmics/corese/releases/latest
+          - version-query: .tag_name | gsub("release-";"")
+          - url-query: '"https://github.com/Wimmics/corese/releases/download/release-" + $version + "/corese-command-" + $version + ".jar"'
```

Comment:
an object is expected for `x-checker-data`  not a list. that's what causing the list validation.